src/main/java/spoon/reflect/visitor/CtInheritanceScanner.java

```diff
@@ -178,23 +195,23 @@ public void scanCtElement(CtElement e) {
 	 * Scans an abstract executable.
 	 */
 	public <R> void scanCtExecutable(CtExecutable<R> e) {
-		scanCtGenericElement(e);
-		scanCtNamedElement(e);
 	}
 
 	/**
 	 * Scans an abstract expression.
 	 */
 	public <T> void scanCtExpression(CtExpression<T> expression) {
```

Comment:
CtExpression inherits from CtCodeElement, CtTypedElement, why we don't need to scan them?